src/Operation/ListCollectionNames.php

```diff
@@ -76,9 +75,7 @@ public function execute(Server $server): Iterator
     {
         return new CallbackIterator(
             $this->listCollections->execute($server),
-            function (array $collectionInfo) {
-                return $collectionInfo['name'];
-            }
+            fn (array $collectionInfo): string => (string) $collectionInfo['name']
```

Comment:
```suggestion
            'strval'
```

Reply:
No objection here unless we wanted to introduce a CollectionInfo type hint to the callable instead.

Reply:
I undid that check as this is in the CallbackIterator: the callback is invoked with two arguments, but `strval` only accepts one.